src/main/java/seedu/duke/commands/Command.java

```diff
@@ -0,0 +1,27 @@
+package seedu.duke.commands;
```

Comment:
It should be better to let Shoco class has 1 Command class everytime as 0..1 is a bit weird.